client/aggregator.go

```diff
@@ -104,7 +104,7 @@ func (c *watchAggregator) startAutoWatch(full bool) {
 			}
 			t := time.NewTimer(c.autoWatchRetry)
 			select {
-			case <-t.C:
+			case <-t.C: //TODO: (dlsniper)HUH??
```

Comment:
Might be worth tracking in an issue?

Reply:
Sorry, that wasn't supposed to make it in the PR.